packages/calling/src/CallingClient/line/index.ts

```diff
@@ -189,6 +190,7 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
       case LINE_EVENTS.UNREGISTERED:
       case LINE_EVENTS.RECONNECTED:
       case LINE_EVENTS.RECONNECTING:
+        this.status = this.registration.getStatus();
```

Comment:
Why did we need this?

Reply:
Removed the status variable itself from the line object as discussed